torch/csrc/autograd/python_torch_functions_manual.cpp

```diff
@@ -491,6 +496,22 @@ static PyObject* THPVariable__disable_functionalization(
   END_HANDLE_TH_ERRORS
 }
 
+static PyObject* THPVariable__sync(
+    PyObject* self,
+    PyObject* args,
+    PyObject* kwargs) {
+  HANDLE_TH_ERRORS
+  static PythonArgParser parser({"_sync(Tensor t)"}, /*traceable=*/true);
+
+  ParsedArgs<1> parsed_args;
+  auto r = parser.parse(args, kwargs, parsed_args);
+  auto self_ = r.tensor(0);
+  TORCH_INTERNAL_ASSERT(at::functionalization::impl::isFunctionalTensor(self_));
+  at::functionalization::impl::sync(self_);
+  Py_RETURN_NONE;
+  END_HANDLE_TH_ERRORS
+}
```

Comment:
Unused?

Reply:
`THPVariable__sync`? I think github just displayed the diff in a not-useful way. `THPVariable__sync` got renamed to `THPVariable__functionalize_has_metadata_mutation` further up, so `THPVariable__sync` got moved here.